src/main/java/lotto/domain/LottoQuantity.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+import lotto.exceptions.InvalidLottoQuantityException;
+
+public class LottoQuantity {
+    private int quantity;
+
+    private LottoQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public static LottoQuantity create(String maybe, Money money) {
+        int quantity = validQuantity(maybe, money);
+        return new LottoQuantity(quantity);
+    }
+
+    private static int validQuantity(String maybe, Money money) {
+        int quantity = validNumber(maybe);
+        validatePositive(quantity);
+        validateEnough(quantity, money);
+        return quantity;
+    }
+
+    private static int validNumber(String maybe) {
+        int quantity;
+        try {
+            quantity = Integer.parseInt(maybe);
```

Comment:
```suggestion
            return Integer.parseInt(maybe);
```

로 바로 응답을 하는 방법도 존재하는데요.
이러한 구현에 정답은 없고 취향에 따라 다르지만 저는 바로 return하는 것을 선호합니다.
그 이유를 생각해보면 try 문에서 로직이 끝나도 메서드의 나머지 내용을 확인하지 않으면 어떠한 값을 응답할지 모르게 되는데요.
지금은 코드가 짧아서 메서드를 파악하기 편하지만, 메서드가 길어진다면 가독성이 떨어진다고 생각했습니다!
혹시 호돌님과 같이 구현했을 때 장점이 있다면 공유해주실 수 있을까요?

Reply:
앗... 딱히 어떤 장점이 있어서라기보단 습관적으로 그렇게 쓴 것 같네요! 저도 바로 `return`하는 것을 선호하긴 하는데 이번에 실수로 친 것 같습니다!